### PR TITLE
galactic to ros2

### PR DESCRIPTION
**Pull request by ahcorde**
# ➡️ Forward port

Port galactic to ros2

Branch comparison: https://github.com/ignitionrobotics/ros_ign/compare/ros2...galactic

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

**Review by mjcarroll — APPROVED** [review]

File: .github/workflows/ros2-ci.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
             ros-distro: "rolling"
           - docker-image: "ubuntu:20.04"
             ignition-version: "fortress"
-            ros-distro: "rolling"
+            ros-distro: "galactic"
```
Comment (chapulina) [inline .github/workflows/ros2-ci.yml]: This branch should be testing `rolling`

**Review by chapulina — CHANGES_REQUESTED** [review]

**Review by chapulina — APPROVED** [review]
LGTM, just pending #219

**Comment by chapulina** [comment]
@ros-pull-request-builder retest this please

**Comment by chapulina** [comment]
No new test failures, merging

**Pull request merged**